tests/test_models/test_engine/test_db_storage.py

```diff
@@ -86,3 +86,46 @@ def test_new(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_save(self):
         """Test that save properly saves objects to file.json"""
+
+
+
+    @unittest.skipIf(models.storage_t != 'db', "not testing file storage")
+    def test_count(self):
+        """ Test for the FS count method
+        """
+        num_states = models.storage.count()
```

Comment:
Don't forget to pass count a class. I'm guessing `State`, so you can check this total later after creating a new one?